homeassistant/components/vizio/__init__.py

```diff
@@ -39,12 +39,9 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     unload_ok = await hass.config_entries.async_unload_platforms(
         config_entry, PLATFORMS
     )
-    # Exclude this config entry because its not unloaded yet
     if not any(
-        entry.state is ConfigEntryState.LOADED
-        and entry.entry_id != config_entry.entry_id
```

Comment:
It looks like this condition is removed

Reply:
yes, it's intentional
The reason why it's no longer needed is explained in https://developers.home-assistant.io/blog/2025/02/19/new-config-entry-states/